internal/flag/flag.go

```diff
@@ -371,7 +371,7 @@ func Now() Bool {
 func NoDeploy() Bool {
 	return Bool{
 		Name:        "no-deploy",
-		Description: "Do not prompt for deployment",
+		Description: "Do not deploy the new app after initial creation and configuration",
```

Comment:
My other idea was:
```suggestion
		Description: "Do not deploy the new app after fly launch creates and configures it",
```

Reply:
Either works for me. I wonder about adding the word "immediately" in there.